### DISCUSSION TITLE
Task Group

### DISCUSSION THREAD

**Discussion by YoSTEALTH** [axboe/liburing#1201] [General]
Is there anything like [Task Group](https://docs.python.org/3/library/asyncio-task.html#task-groups) in liburing/io_uring? For example being able to create child-task and waiting for those child-task to finish before continuing.

**Comment by isilence** [comment]
It's an abstraction of that particular library, there is no place for it in io_uring. However you can wait for threads / processes with `IORING_OP_WAITID`. Similar to `waitid(2)`, but io_uring and async.
